zingolib/src/config.rs

```diff
@@ -644,57 +624,25 @@ mod tests {
 
     use crate::wallet::WalletSettings;
 
-    /// Validate that the `load_clientconfig` function creates a valid config from an empty uri
-    #[tokio::test]
-    async fn test_load_clientconfig() {
-        rustls::crypto::ring::default_provider()
-            .install_default()
-            .expect("Ring to work as a default");
-        tracing_subscriber::fmt().init();
-
-        let valid_uri = crate::config::construct_lightwalletd_uri(Some(String::new()));
-
-        let temp_dir = tempfile::TempDir::new().unwrap();
-
-        let temp_path = temp_dir.path().to_path_buf();
-
-        let valid_config = crate::config::load_clientconfig(
-            valid_uri.clone(),
-            Some(temp_path),
-            crate::config::ChainType::Mainnet,
-            WalletSettings {
-                sync_config: pepper_sync::config::SyncConfig {
-                    transparent_address_discovery:
-                        pepper_sync::config::TransparentAddressDiscovery::minimal(),
-                    performance_level: pepper_sync::config::PerformanceLevel::High,
-                },
-                min_confirmations: NonZeroU32::try_from(1).unwrap(),
-            },
-            1.try_into().unwrap(),
-            "".to_string(),
-        );
-
-        assert!(valid_config.is_ok());
-    }
-
     #[tokio::test]
     async fn test_load_clientconfig_serverless() {
         rustls::crypto::ring::default_provider()
             .install_default()
             .expect("Ring to work as a default");
         tracing_subscriber::fmt().init();
 
-        let valid_uri = crate::config::construct_lightwalletd_uri(Some(
+        let valid_uri = crate::config::parse_indexer_uri(
             crate::config::DEFAULT_LIGHTWALLETD_SERVER.to_string(),
-        ));
+        )
```

Comment:
it seems you have removed a load config test that should contain this URI (and incorrectly tested serverless previously) but now have added a serverless test which creates a config *wtih* a server?

can we call this test `load_clientconfig` and duplcate it and call it `load_clientconfig_serverless` with a `None` in the URI field and change the assertions accordingly?